intergov/channels/http_api_channel.py

```diff
@@ -128,15 +130,29 @@ def _get_cached_cognito_jwt(self):
 
     def _issue_cognito_jwt(self):
         # see _get_cached_cognito_jwt descr about the design concerns
-        OAUTH_CLIENT_ID = env("IGL_OAUTH_CLIENT_ID", default=None)
-        OAUTH_CLIENT_SECRET = env("IGL_OAUTH_CLIENT_SECRET", default=None)
-        OAUTH_SCOPES = env("IGL_OAUTH_SCOPES", default=None)
+        new_style_details = self.CONFIG.get("ChannelAuthDetails")
+        if new_style_details:
```

Comment:
do we need to support both styles? can we just drop the old one?

Reply:
we will drop it once @roleyfoley confirms that it's not used anymore (in some days)

Reply:
I'll add the config for the new format now, feel free to remove it and it can just be broken until this has been merged in